eng/pipelines/runtime.yml

```diff
@@ -500,6 +500,20 @@ jobs:
           eq(dependencies.checkout.outputs['SetPathVars_libraries.containsChange'], true),
           eq(variables['isFullMatrix'], true))
 
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
+    buildConfig: Release
+    platforms:
+    - Linux_x64
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    jobParameters:
+      isFullMatrix: false
```

Comment:
you shouldn't need this... as `helix-queues-setup` shouldn't come into place since `runTest` is false.

Reply:
anyway, please follow what we do at other places: `${{ variables.isFullMatrix }}`.